### DISCUSSION TITLE
[STANDALONE] project template

### DISCUSSION THREAD

**Discussion by Myziak** [enable3d/enable3d#201] [General]
hi, i wanted start my journey with enable3d - but i cant see the project template (only phaser-exntension, and three ext).
Where i can find it?

**Comment by vitalspace** [comment]
Hmm

https://enable3d.io/examples/standalone-mode.html

**Reply by Myziak** [reply]
project template have local enviroment, which is connected with TS, and ready module bundler ;)

Look this: 
![image](https://user-images.githubusercontent.com/75075069/134135504-42f13dbe-601b-4a1d-a6fa-75581aa705be.png)

We have ready enviro for phaser and three.js+ammo, but not for main version od library.

**Comment by yandeu** [comment]
I have just updated the [website](https://enable3d.io/) and the [enable3d template](https://github.com/enable3d/threejs-ammojs-webpack-example).
